tests/storage/test_online_resize.py

```diff
@@ -296,9 +296,8 @@ def test_disk_expand_then_clone_fail(
             sleep=TIMEOUT_5SEC,
             func=lambda: dv.instance.status.conditions,
         ):
-            if any(
-                "The clone doesn't meet the validation requirements:"
-                " target resources requests storage size is smaller than the source" in condition["message"]
+            if sample and any(
```

Comment:
why iterate over all conditions? i'd assume this message belongs to a specific condition type
then you can use `dv.get_condition_message`